src/test/common/steps/pages.js

```diff
@@ -82,6 +84,7 @@ class Pages {
     this.iLiveInScotland = new ILiveInScotland(this.driver)
     this.emailAddress = new EmailAddress(this.driver)
     this.unsuccessfulApplication = new UnsuccessfulApplication(this.driver)
+    this.termsAndConditions = new TermsAndConditions(this.driver)
     // NOTE: This map should contain all page objects, and not the Generic Page as this doesn't itself represent a page
     this.allPages = [this.overview, this.enterName, this.enterNino, this.enterDOB, this.areYouPregnant, this.cardAddress, this.phoneNumber,
```

Comment:
You need to add this page into the `allPages` list too.

Reply:
done, thanks for spotting :)